### ISSUE TITLE
Image Loading Issue in the Portal - Critical Issue with Image file loading

### ISSUE THREAD

**Issue by bikirandas** [bikirandas/OmexOnline#2]
After Loging in to the Omex portal. Visiting the Profile page doesnot shows the profile pic even though the pic is uploaded.

**Comment by bikirandas** [comment]
Identified the issue - Issue is with generating the image src in the views.py  - generated src is _/users/rony/f4098984_Kgo0lqi.jpg_     - But when the user_profile page is loaded the src gets updated to /users/rony/users/rony/f4098984_Kgo0lqi.jpg 

the above is fixed in the views.py using the following code
```
def user_profile(request, username):
    image = request.user.profile.img
    image = str(image)
    image = image.strip('users/%s' % username)
    print(image)
    return render(request, 'user_profile.html', {'username': request.user.username,
                                                 'image': image})
```
But the required SRC should be   '../users/rony/f4098984_Kgo0lqi.jpg'

**Comment by bikirandas** [comment]
Error - :WARNING:"GET /users/rony/f4098984_Kgo0lqi.jpg HTTP/1.1" 404 2665 

when browsing profile page.

the GET Link generated is correct - but pic is not opening in the webpage.

**Comment by bikirandas** [comment]
https://stackoverflow.com/questions/55218329/django-image-not-getting-displayed-http-1-1-404-2665-warning

**Comment by bikirandas** [comment]
The above two commits on 18th march are with respect to change in the location of the picture to be displayed in the webpage. Change was confined to [user_profile.html](https://github.com/bikirandas/OmexOnline/blob/master/Accounts/templates/user_profile.html)

**Comment by bikirandas** [comment]
Tested by changing to  `<img src="..{{ image }}" alt="Profile Picture">`

Server Log - 

> 18.03.19 23:52:17:WARNING:"GET /users/rony/..f4098984_Kgo0lqi.jpg HTTP/1.1" 404 2671

**Comment by bikirandas** [comment]
Tried with adding media_url and media_root in settings.py -- further templates context processor added with 'django.template.context_processors.media' -- 
_issue still persists_

**Comment by bikirandas** [comment]
@sudheersingampalli hey Bro can you help me with this issue.

**Comment by bikirandas** [comment]
Issue is resolved with the commit [a0655e9](https://github.com/bikirandas/OmexOnline/commit/a0655e9d0c97218862ed5fd9593de4f75fe3258e)

**Comment by sudheersingampalli** [comment]
hi bro..lets connect on whatsapp whenever there is an issue..these mail
keep going into the forum tab and I'll miss them most of the time.

Thanks
